src/frontend/apps/desk/src/pages/teams/[teamId]/members/add.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { ModalAddMembers } from '@/features/members/components/ModalAddMembers';
+import { NextPageWithLayout } from '@/types/next';
+
+import { TeamDetailLayout } from '..';
+
+const Page: NextPageWithLayout = () => {
+  return null;
+};
+
+Page.getLayout = function getLayout() {
+  return (
+    <TeamDetailLayout>
+      {(team) => <ModalAddMembers teamId={team.id} />}
+    </TeamDetailLayout>
```

Comment:
I feel it's a bit overkill having a dedicated `url` to add members. If we look at other services (GitHub, Trello, drive, ...) The pop-in to add/invite users, is not linked to any `url`. What motivated creating a subpage?

Reply:
I removed the commit and adapted the tests:
https://github.com/numerique-gouv/people/pull/125/commits/f4e8b49b97b287412c6fc3021e654e83c618283f#diff-2969e8751c72b7ce75b2eaac6c2a3899d4b34f439a128a13b48fb53737f38ad9R1